src/pages/bookPage.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Book from "../components/book";
+import Header from "../components/header";
+
+const bookPage = () => (
+  <main>
+    <Header />
+    <Book category="Action" author="Suzanne Collins" title="The Hunger Games" />
+    <Book category="Science Fiction" author="Frank Herbert" title="Dune" />
+    <Book
+      category="Economy"
+      author="Suzanne Collins"
+      title="Capital in the Twenty-First Century"
```

Comment:
- [ ] Kindly, make sure that you're calling just once the function `Book`. for now, your project looks like this: 
![image](https://user-images.githubusercontent.com/71874425/166120091-d4105dbf-b201-4173-8447-f971780edab4.png)
But, you should have just once the form for adding books. 
Kindly, fix these issues and then ask for a new review.